pkg/splunk/splunk.go

```diff
@@ -53,41 +52,38 @@ type SearchResults struct {
 	Preview     bool                `json:"preview"`
 	Results     []SearchResult      `json:"results"`
 	Highlighted map[string]string   `json:"highlighted"`
-	//Fields    []map[string]string `json:"fields"`
 }
 
+type SplunkServer config.SplunkConfig
```

Comment:
really nitpicky thing - we can just call this `Server`, as the package name clarifies that it's for splunk.

Reply:
Done